### PR TITLE
Setup express app, pug, and basic layout

### PR DESCRIPTION
**Pull request by lumodon**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const app = express()
+const pug = require('pug')
+const normalized = require('express-normalized')
```
Comment (nicosesma) [inline app.js]: It's interesting that you're using express-normalized

File: public/stylesheets/style.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  font-size: 2em;
+  padding: 10px;
+  text-align: center;
+}
+.interface {
+  text-align: center;
+  padding-bottom: 5em;
+}
+.interface * {
+  padding: 15px;
```
Comment (nicosesma) [inline public/stylesheets/style.css]: does this override the padding-bottom defined earlier?

File: views/index.pug [hunk 3]
```diff
@@ -0,0 +1,21 @@
+doctype html
```
Comment (nicosesma) [inline views/index.pug]: everything seems to be clearly defined seems great

File: public/stylesheets/style.css (hunk 2 above)
Comment (lumodon) [inline public/stylesheets/style.css]: This one enables the children to have a different padding than the container itself.

File: app.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+// app.set('/', __dirname + '/views') //double check this
+app.set('views', __dirname + 'views')
+
+// app.use('/', index)
+app.use(express.static('public'))
+app.use(normalized())
+
+app.get('/', function(request, response) {
+  // response.send('Hello Tacos!')
+  // response.render(index)
```
Comment (jason00111) [inline app.js]: Some of these comments can probably be removed

**Pull request merged**